script/before_backstop.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+
+layout ||= ARGV[0] || nil
```

Comment:
This line is strange - `layout = ARGV[0]` should work identically

Reply:
...I think that's an artifact of this being basically before_wraith renamed, which was _also_ used in another script. or at least the `||=` is. idk about the `|| nil` xD